### DISCUSSION TITLE
Issue with "Common R markdown syntax cheatsheet"

### DISCUSSION THREAD

**Discussion by kfijan** [jtr13/cc21fall1#89] [Q&A] [answered]
I believe #19 and #20 in our Book is one submission split into two pages because of a header title midway through the file. (Please see 19 & 20 in our book website).  Titles are: "Common R markdown syntax cheatsheet" and "Header". 
@s10singh97 @Shruti-Kaushal

**Comment by s10singh97** [comment, marked as answer]
Rectified the issue.
